SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -434,6 +435,47 @@ FHandoverChangeState USpatialActorChannel::CreateInitialHandoverChangeState(cons
 	return HandoverChanged;
 }
 
+void USpatialActorChannel::UpdateVisibleComponent(AActor* InActor)
+{
+	// Make sure that the Actor is not always relevant.
+	if(InActor->bAlwaysRelevant)
+	{
+		return;
+	}
+
+	// Make sure that the actor is not PlayerController, GameplayDebuggerCategoryReplicator and GameMode.
+	APlayerController* PC = Cast<APlayerController>(InActor);
+	if(IsValid(PC))
+	{
+		return;
+	}
+
+	FString ActorsName = InActor->GetName();
+	if(ActorsName.Contains(TEXT("GameplayDebuggerCategoryReplicator")))
```

Comment:
I think we may have to rethink this somehow

Reply:
Any suggestions, casting it requires the module to be included which might be excessive

Reply:
:( I just don't like this external class inspection to mirror the NetRelevancy rules from native.
But I can't think of a better solution.